scripts/indexes.js

```diff
@@ -38,6 +38,8 @@ db.hosts.ensureIndex({ "status" : 1, "create_time": 1, "termination_time" : 1, "
 db.hosts.ensureIndex({ "running_task": 1}, {sparse: true, unique: true})
 db.hosts.createIndex({ "last_bv": 1, "last_group": 1, "last_project": 1, "last_version": 1, "status": 1, "last_task": 1 }, {background: true})
 db.hosts.createIndex({ "running_task_bv": 1, "running_task_group": 1, "running_task_project": 1, "running_task_version": 1, "status": 1, "running_task": 1 }, {background: true})
+db.hosts.createIndex({ "container_id": 1 })
```

Comment:
At merge, you should create a JIRA ticket in the MCI project to build these indexes